tests/e2e/util.go

```diff
@@ -6119,8 +6127,15 @@ func writeDataOnRawBlockVolume(ns string, podName string, devicePath string, tes
 	_, err = framework.RunKubectl(ns, cmd...)
 	framework.ExpectNoError(err, fmt.Sprintf("failed to write testdata inside the pod: %q", podName))
 
-	cmd = []string{"exec", podName, "--namespace=" + ns, "--", "/bin/sh", "-c",
-		fmt.Sprintf("/bin/dd if=/tmp/data_to_write of=%v", devicePath)}
+	// If dataSize is given, fill the device with random data of size given.
+	// Otherwise write the testData given.
+	if dataSizeInMB != 0 {
```

Comment:
in both cases below we don't know what was written to the device right?

Reply:
if testData input is given, we write the same data on device but if only dataSize is specified, we write random data.